splunktaucclib/rest_handler/util.py

```diff
@@ -95,3 +98,44 @@ def remove_http_proxy_env_vars():
             del os.environ[k]
         elif k.upper() in os.environ:
             del os.environ[k.upper()]
+
+
+def get_proxy_uri(proxy) -> Optional[str]:
```

Comment:
Let's at least say that `proxy` is `Dict[str, Any]`

Reply:
Updated the code